toolchain/lex/tokenized_buffer.cpp

```diff
@@ -254,7 +256,7 @@ auto TokenizedBuffer::PrintToken(llvm::raw_ostream& output_stream,
   // justification manually in order to use the dynamically computed widths
   // and get the quotes included.
   output_stream << llvm::formatv(
-      "    { index: {0}, kind: {1}, line: {2}, column: {3}, indent: {4}, "
+      "  - { index: {0}, kind: {1}, line: {2}, column: {3}, indent: {4}, "
```

Comment:
What's the purpose of the `-`?

Reply:
This is the alternate YAML sequence syntax. See https://www.tutorialspoint.com/yaml/yaml_sequence_styles.htm for examples.